data/ontime/download.py

```diff
@@ -45,17 +48,6 @@ def download(startyear, startmonth, endyear, endmonth):
         else:
             print("Already having " + basename)
 
-def main():
```

Comment:
are you sure this code is no longer needed?

Reply:
Yes. Basically this part adds `--cipher 'DEFAULT:!DH'` option to `wget` if it detects the system is ubuntu20.04. With the `--no-check-certificate` option I don't think this is needed anymore.

Reply:
> I assume this works with older versions of Ubuntu too?

Yes, they should work with ubuntu 18.04 as well.

There is another fix for the wget problem though, which is to check in the [entrust root certificate](https://web.entrust.com/root-certificates/entrust_l1k.cer) to our repo, and then replace the `--no-check-certificate` option with `--ca-certificate=entrust_l1k.cer` so it will use the provided certificate. It might be safer that way?